app/bingey-ui/src/utils/filter.js

```diff
@@ -0,0 +1,23 @@
+export const filter = (values, filters) => {
+  return values?.filter((value) => isValid(value, filters));
+};
+
+const isValid = (value, filters) => {
+  let filter;
+
+  for (let i = 0; i < filters.length; i++) {
```

Comment:
filters.map should be used here no reason to have a for loop

Reply:
If a filter fails, this will not check further fields